src/components/site/PostToc.tsx

```diff
@@ -60,12 +64,17 @@ function renderItems(items: TocResult["map"], activeId: string, prefix = "") {
               child.children[0].children?.[0]?.children ||
               child.children[0].children
             let content = "",
-              isInlineMath = false
+              isInlineMath = false,
```

Comment:
I personally think that defining separately may be a more maintainable approach.